.gitignore

```diff
@@ -0,0 +1,8 @@
+# OS metadata
+.DS_Store
+Thumbs.db
```

Comment:
The better place for this is in the global git configuration, so these rules aren't exported to all open source projects.